app/views/users/show.html.erb

```diff
@@ -0,0 +1,103 @@
+<div class="container">
+
+  <ul class="nav nav-pills mb-3" id="pills-tab" role="tablist">
+    <li class="nav-item">
+      <a class="nav-link active" id="pills-home-tab" data-toggle="pill" href="#pills-home" role="tab" aria-controls="pills-home" aria-selected="true">Bookings</a>
+    </li>
+    <li class="nav-item">
+      <a class="nav-link" id="pills-profile-tab" data-toggle="pill" href="#pills-profile" role="tab" aria-controls="pills-profile" aria-selected="false">Listings</a>
+    </li>
+  </ul>
+
+  <div class="tab-content" id="pills-tabContent">
+    <div class="tab-pane fade show active blue-background" id="pills-home" role="tabpanel" aria-labelledby="pills-home-tab">
+      <div class="bookings">
+
+        <div class="card profile-card" >
+          <div class="card-header">
+            <h1>Welcome <%= @user.first_name %></h1>
+          </div>
+          <div class="card-body d-flex justify-content-around align-items-center">
+            <div class="user-info">
+              <h5 class="card-title">Passionate gardener living in Beirut</h5>
+              <p class="card-text"> Contact information:</p>
+              <ul>
+                <li> EMAIL: <%= @user.email %></li>
+                <li> NAME: <%= @user.first_name %> <%= @user.last_name %></li>
+              </ul>
+            </div>
+            <%= cl_image_tag @user.avatar.key, alt: "user-avatar", width: 200, height: 200, crop: :fill %>
```

Comment:
we need a condition here. A user might not upload an image and in that case this line will throw an error. So you would need something like this 👇 
```erb
<% if @user.avatar.attached? %>
  <%= cl_image_tag @user.avatar.key, alt: "user-avatar", width: 200, height: 200, crop: :fill %>
<% end %>
```

You can have an `else` as well to display a default avatar if the user doesn't upload an image

Reply:
Thank you will add this